GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -277,16 +281,39 @@ bool ExporterHelper::ExportCordovaFiles(const gd::Project &project,
           .FindAndReplace("<!-- GDJS_ICONS_ANDROID -->", makeIconsAndroid())
           .FindAndReplace("<!-- GDJS_ICONS_IOS -->", makeIconsIos());
 
-  if (!project.GetAdMobAppId().empty()) {
-    str = str.FindAndReplace(
-        "<!-- GDJS_ADMOB_PLUGIN_AND_APPLICATION_ID -->",
-        "<plugin name=\"cordova-plugin-admob-free\" spec=\"~0.21.0\">\n"
-        "\t\t<variable name=\"ADMOB_APP_ID\" value=\"" +
-            project.GetAdMobAppId() +
-            "\" />\n"
-            "\t</plugin>");
+  gd::String plugins = "";
+
+  for(gd::String const extensionName : project.GetUsedExtensions()) {
```

Comment:
Sorry for the confusion about GetUsedExtensions, this should be probably marked as deprecated/removed.

The idea is that we don't even need to know which extension the project relies on, because the extension action/condition/object are exposing all that is needed to compile the game (mainly: the include files).
This is unfortunately not working super well for your use case. 

For now, I would suggest that you iterate on *all* the extensions of the `gd::Platform`, using `GetAllPlatformExtensions` (see https://docs.gdevelop-app.com/GDCore%20Documentation/classgd_1_1_platform.html#a077aa5264fa2a6ebf82af5a1eb3a7fd2).
It's not optimal, because if an extension is loaded, it would then always write its cordova dependencies even if not used in the project (which could result in AdMob being installed even when not asked). That's not something that we want! But at least you can continue working on this without being blocked, and we can think later how to improve this.

(It's not super easy to fix because:
- I don't like the old approach of having a vector of extensions used by the project, because you need to manually keep it updated. How do you even remove an extension from it?
- the alternative is to automatically generate it when needed. But for this you need to browse the whole project to identify the objects/behavior/actions/conditions/effects/events used and identify their parent extension. It's doable and surely the way to go, but I try to limit classes that have to do work on the whole project).